base/gcd.h

```diff
@@ -0,0 +1,40 @@
+// LAF Base Library
+// Copyright (c) 2019 Igara Studio S.A.
+//
+// This file is released under the terms of the MIT license.
+// Read LICENSE.txt for more information.
+
+#ifndef BASE_GCD_H_INCLUDED
+#define BASE_GCD_H_INCLUDED
+#pragma once
+
+namespace base {
+
+  template<typename T>
+  T gcd(T a, T b) {
+    T output = 1;
+    a = a<0? -a : a;
+    b = b<0? -b : b;
+    if (a > 1 && b > 1) {
+        if (a == b)
```

Comment:
Use indent level to 2 always, it looks like lines 15-18 use 2 whitespace for indentation, and now line 19 use four whitespaces. Use 2 whitespaces always.